app/controllers/admin/job_openings_controller.rb

```diff
@@ -1,12 +1,11 @@
 class Admin::JobOpeningsController < AdminController
+  load_and_authorize_resource
 
   def index
     @job_opening = JobOpening.new
-    @job_openings = JobOpening.all
   end
 
   def create
-    @job_opening = JobOpening.new(job_opening_params)
```

Comment:
MAGIC! cancancan knows that you want to make a new job_opening with the params because of naming.